contrib/gitian-build.py

```diff
@@ -31,6 +31,10 @@ def setup():
         make_image_prog += ['--lxc']
     subprocess.check_call(make_image_prog)
     os.chdir(workdir)
+    if args.is_bionic:
+        subprocess.check_call(['sudo', 'sed', '-i', 's/lxcbr0/br0/', '/etc/default/lxc-net'])
+        print('Reboot is required')
+        exit(0)
```

Comment:
Shouldn't this be only executed for lxc (and not docker/kvm)?

Reply:
Yes.